### ISSUE TITLE
Keep alive not working in chromium browsers

### ISSUE THREAD

**Issue by dixsie** [openbmc/webui-vue#87]
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/45175

https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/47767

https://discord.com/channels/775381525260664832/776550056391606352/896145249217417226

**Comment by jmbills** [comment]
I brought this up in Discord and it was suggested that this is not related to keep-alive but rather to a timer_queue limitation.  I tested the recommended changes to the timer_queue in bmcweb and it seems to resolve the issue on my system.

Here is the info from Discord:
edtanous — Yesterday at 1:18 PM
keepalive should have no effect on the logged in/logged out status of a user
whether a user is logged in is based on their cookie, keepalive is on a per-connection basis
edtanous — Yesterday at 1:32 PM
I think @Sunitha Harish was looking into something where we weren't cleaning up the timer slots after a change James made a while back if you're doing a relatively high number of http calls..  If you quadruple the size of this, does your problem go away? 
https://github.com/openbmc/bmcweb/blob/60c922dfacd5d1aeec8789e03edc91b47f4a6661/http/timer_queue.hpp#L18 
Also, as we're pushing more and more traffic, we should probably decrease timerQueueTimeoutSeconds from 5 seconds down to 1.
that'll help as well
GunnarM — Yesterday at 4:28 PM
As a data point, moving maxSize to 300 did fix the errors @Sunitha Harish was seeing with a higher number of http calls 
https://github.com/openbmc/bmcweb/blob/60c922dfacd5d1aeec8789e03edc91b47f4a6661/http/timer_queue.hpp#L18
jmbills — Today at 8:47 AM
I tried both maxSize at 1000 and timerQueueTimeoutSeconds at 1 independently, and each of them appears to resolve the issue on my system.  I will push these changes for review.  How big do we want the maxSize to be?
